bytecomp/bytepackager.ml

```diff
@@ -21,6 +21,11 @@ open Instruct
 open Cmo_format
 module String = Misc.Stdlib.String
 
+let rec rev_append_map f l rest =
```

Comment:
Maybe this function could be moved to `Misc.Stdlib.List`, as it could be useful outside this file?